### PR TITLE
Show all team-members [redux + redux-sagas]

### PR DESCRIPTION
**Pull request by jellebouwman**
Added reducers, selectors and edited the TeamMemberOverview component to accommodate dynamically loading all saved Team Members

### CODE REVIEW

File: components/team-member/TeamMember.jsx [hunk 1]
```diff
@@ -23,35 +39,40 @@ class TeamMember extends React.Component {
...
+              {currentEmployer}
             </p>
             <p className="team-member__subtitle">
               Current employer
             </p>
           </div>
           <div className="team-member__starting-date">
             <p className="team-member__title">
-              February 2018
+              {new Date(startingDate)
```
Comment (RikFrieling) [inline components/team-member/TeamMember.jsx]: Maybe transfer this as a function to your services?

Comment (jellebouwman) [reply components/team-member/TeamMember.jsx]: When it becomes clear on what the startingDate is based on (maybe another input in a second tab, or from the first time-entry: like you suggested), i'll consider abstracting this to a generic service.

For now I think I agree with you agreeing on this solution working for this situation

File: components/team-member/TeamMember.jsx [hunk 2]
```diff
@@ -13,8 +24,13 @@ class TeamMember extends React.Component {
...
   render() {
     const { isTeamMemberExpanded } = this.state;
+    const {
+      avatar, currentEmployer, employeeNumber,
+      firstName, jobTitle, lastName,
+      startingDate
+    } = this.props;
     return (
-      <div className="team-member-wrapper">
+      <div className="team-member__wrapper">
```
Comment (boris-arkenaar) [inline components/team-member/TeamMember.jsx]: Use the `team-member` css class for the root component.

File: sagas/team-members.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import {
+  REQUEST_TEAM_MEMBERS,
+  requestTeamMembersSucces
+} from '../ducks/team-members';
+
+import {
+  getTeamMembers
+} from '../services/fetch-team-members/fetch-team-members';
+
+function* onGetTeamMembers() {
```
Comment (boris-arkenaar) [inline sagas/team-members.js]: Sagas don't use the prefix `on`.

File: ducks/team-members.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    default:
+      return state;
+  }
+}
+
+export const requestTeamMembers = () => ({
+  type: REQUEST_TEAM_MEMBERS
+});
+
+export const requestTeamMembersSucces = teamMembers => ({
```
Comment (boris-arkenaar) [inline ducks/team-members.js]: `success` is with two `s`s

File: services/fetch-team-members/fetch-team-members.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+const api = 'http://localhost:3001/api/';
+
+export const getTeamMembers = () => (
+  fetch(`${api}teamMembers/`)
```
Comment (boris-arkenaar) [inline services/fetch-team-members/fetch-team-members.js]: Use dash notation in url's; `/api/team-members/`.

File: components/team-member-overview/TeamMemberOverview.jsx [hunk 6]
```diff
@@ -48,9 +67,19 @@ class TeamMemberOverview extends React.Component {
           </button>
         </div>
         <div className="team-member__content-wrapper">
-          <TeamMember />
-          <TeamMember />
-          <TeamMember />
+          {teamMembers.map(teamMember => (
+            <React.Fragment key={teamMember.id}>
```
[the remark this answers is not in the document]
Comment (jellebouwman) [reply components/team-member-overview/TeamMemberOverview.jsx]: Like discussed, it is necessary in this instance.

**Pull request merged**